### DISCUSSION TITLE
Is it possible to segment choices?

### DISCUSSION THREAD

**Discussion by laur89** [davatorium/rofi#2047] [General]
Given following 4 options:
```sh
$ echo -e "Option #1\nOption #2\nOption #3\nOption #4" | rofi -dmenu
```

is there a way to visually separate options 1-2 & 2-3 into two separate segments/sections? Maybe draw a thicker line between them or alternate background colors?

**Comment by DaveDavenport** [comment]
No.. You could try putting a 'special' row in between..  see the rofi-dmenu/rofi-script manpage.
